### ISSUE TITLE
Cannot access inside beforeEach block at root level

### ISSUE THREAD

**Issue by noahprince22** [stalniy/bdd-lazy-var#45]
Trying to use a variable, like so

```
def('hello', () => 'hello')

beforeEach(() => {
  const hello = $hello;
})

```

Fails with `It looks like you are trying to evaluate "hello" too early.`

**Comment by stalniy** [comment]
Hello,

What framework do you use (name + version)?

**Comment by stalniy** [comment]
Fixed in `2.2.4`
